bigquery_etl/stripe/__init__.py

```diff
@@ -52,30 +52,21 @@ def _get_report_rows(
         yield from sys.stdin.buffer
     else:
         stripe.api_key = api_key
-        report_type_ = stripe.reporting.ReportType.retrieve(report_type)
         parameters: Dict[str, Any] = {"columns": columns}
         if after_date:
-            parameters["interval_start"] = (int(after_date.timestamp()),)
-            if parameters["interval_start"] < report_type_.data_available_start:
-                start_limit = datetime.utcfromtimestamp(
-                    report_type_.data_available_start
-                )
-                raise click.ClickException(
-                    f"Report data not available for {after_date:%F},"
-                    f" data is only available from {start_limit:%F}"
-                )
+            parameters["interval_start"] = int(after_date.timestamp())
         if before_date:
-            parameters["interval_end"] = (int(before_date.timestamp()),)
-            if parameters["interval_end"] > report_type_.data_available_end:
-                end_limit = datetime.utcfromtimestamp(report_type_.data_available_end)
-                raise click.ClickException(
-                    f"Report data not available up to {before_date:%F},"
-                    f" data is only available up to {end_limit:%F}"
-                )
-        run = stripe.reporting.ReportRun.create(
-            report_type=report_type,
-            parameters=parameters,
-        )
+            parameters["interval_end"] = int(before_date.timestamp())
+
+        try:
+            run = stripe.reporting.ReportRun.create(
+                report_type=report_type,
+                parameters=parameters,
+            )
+        except stripe.error.InvalidRequestError as e:
+            # Wrap exception to hide unnecessary traceback
+            raise click.ClickException(str(e))
```

Comment:
Do the Stripe error messages make it clear enough that they're Stripe-related?

If not, it may be helpful to preface the error:
```suggestion
            raise click.ClickException(f"Error creating Stripe report run:  {e}")
```

Reply:
errors look like ```Error: Request req_aCxtk53Mm7oJ1O: Data for the report type payout_reconciliation.itemized.5 is only available through 2022-02-25 12:00:00 UTC; you requested `interval_end` = 2022-03-01 00:00:00 UTC.```, so I think they're thorough enough not to need prefacing.